Automattic/woocommerce-payments
# Remove `OrderMetaHelper` and use multi-currency reference instead of global function in `RestController`

#### Changes proposed in this Pull Request

This PR updates the `multi-currency-v2` feature branch to use the MCCY internal reference instead of its global function. This avoids problems with V1 and V2 installed both at the same time.

This PR also removes the `OrderMetaHelper` class because it's no longer used (p1728333328057959-slack-CB37YNEH1).

#### Testing instructions

- Switch to this branch and run `npm run start`.
- Smoke test multi-currency settings, single currency settings, placing an order.
- Ensure the order meta tool is no longer available when using the `&wcpay_mc_meta_helper=1` parameter in the edit order page (https://github.com/Automattic/woocommerce-payments/pull/6936).


-------------------


**Post merge**
